src/js/populate_question_form.js

```diff
@@ -0,0 +1,49 @@
+"use strict";
+
+/* global wordle_globals */
+
+var inverted_wordles = {};
+
+// Populate the data on the answer question page
+inverted_wordles.setFormData = function (response, options) {
+    response.json().then(questionFile => {
+        inverted_wordles.setEscapedContent(options.selectors.questionTitle, questionFile.title);
+        inverted_wordles.setEscapedContent(options.selectors.entryMaxLength, questionFile.entryMaxLength);
+
+        let entriesHtml = "";
+        for (let i = 0; i < questionFile.entries; i++) {
+            entriesHtml += "<input type=\"text\" autocomplete=\"false\" ";
+            if (i === 0) {
+                entriesHtml += "autofocus";
+            }
+            entriesHtml += " name=\"answer\" maxlength=\"" + questionFile.entryMaxLength + "\" placeholder=\"Enter a word or a phase\">";
+        }
+
+        document.querySelector(options.selectors.entryArea).innerHTML = entriesHtml;
+    });
+};
+
+inverted_wordles.setEscapedContent = function (selector, content) {
+    const escapeHTML = function (html) {
```

Comment:
This may as well be a top-level function. Do reference the stackOverflow posting where you got the algorithm from : P